SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security
+
+The Vega team and community take security bugs in Vega seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+To report a security issue privately, please use the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/vega/vega/security/advisories/new) tab.
+
+A Vega maintainer will send a response indicating next steps in handling your report. After the initial reply, the team will keep you informed of the progress towards a fix and announcement, and may ask for additional information or guidance.
+
+You can also report a vulnerability through the [npm contact form](https://www.npmjs.com/support) by selecting "I'm reporting a security vulnerability".
+
+## Languages
+
+Communications should be in English.
```

Comment:
I think we can delete that since we already have that for the org. If we ever get a request not in English, we can respond.